Checkout/Checkout/Source/Model/Card+Scheme.swift

```diff
@@ -19,7 +19,9 @@ extension Card {
   ///   - Maestro
   ///   - Mastercard
   ///   - Visa
-  public enum Scheme: String, CaseIterable, CustomStringConvertible {
+  public enum Scheme: CaseIterable, Equatable, Hashable {
```

Comment:
```suggestion
@frozen public enum Scheme: CaseIterable, Equatable, Hashable {
```

Reply:
> I am not a fan of using frozen as this may as well change and it would become a breaking change for consumers. Would work if we were defining comparisons (< = >) or other things that would never change, but card schemes do come and go (example: we're looking at CB)